src/scripts/main.js

```diff
@@ -1,3 +1,51 @@
 'use strict';
-
 // write your code here
+
+const population = document.querySelectorAll('.population');
+
+function populationNum(currentPupulation) {
+  return +currentPupulation.innerText.replaceAll(',', '');
+}
+
+const populationArray = [...population].map(populationNum);
+const totalPopulation = populationArray.reduce(
+  (sum, currentValue) => sum + currentValue);
+const avgAmountPopulation = totalPopulation / populationArray.length;
+
+const spanCalculateItTotal = document.querySelector('.total-population');
+const spanCalculateItAverage = document.querySelector('.average-population');
+
+function populationStr(num) {
+  const str = num.toString();
+
+  let result = '';
+  let start;
+
+  if (str.length % 3 === 0) {
+    start = 1;
+  };
+
+  if (str.length % 3 === 1) {
+    start = 3;
+  };
+
+  if (str.length % 3 === 2) {
+    start = 2;
+  };
```

Comment:
If you use this for adding commas - I propose you better way (also you can replace all `.` to `,`
![image](https://user-images.githubusercontent.com/56752562/220323398-8edfdd76-a14e-41ed-85a8-c0a5cc8969a7.png)